src/uct/ib/efa/ib_efa_dv.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCT_IB_EFA_DV_H_
+#define UCT_IB_EFA_DV_H_
+
+#ifndef UCT_IB_EFA_H_
+# error "Never include <uct/ib/efa/ib_efa_dv.h> directly; use <uct/ib/efa/ib_efa.h> instead."
+#endif
+
+#include <ucs/type/status.h>
+#include <uct/ib/base/ib_device.h>
+#include <infiniband/efadv.h>
+
+
+typedef struct efadv_device_attr uct_ib_efadv_attr_t;
```

Comment:
why needed?

Reply:
I didn't want to use efadv_device_attr directly in UCT. I want everything exposed from ib_efa.h (or ib_efa_dv.h) to be under uct_ib_ namespace.

Reply:
well, in other places in ib transport we just use "struct XX" directly, and do not (usually) bother to typedef stuff from included headers

Reply:
OK, I will do the same here too.